
[2022-06-19T13:57:14.052] [ERROR] default - Server error Error: listen EADDRINUSE: address already in use :::8080
[2022-06-19T13:59:06.177] [ERROR] default - CustomError {
  estado: 404,
  descripcion: 'Document not found with that {"id":"A003"}',
  detalles: undefined
}
[2022-06-19T14:01:21.523] [ERROR] default - CustomError {
  estado: 404,
  descripcion: 'Document not found with that {"id":"A003"}',
  detalles: undefined
}
[2022-06-19T16:36:23.052] [ERROR] default - ReferenceError: productos is not defined
    at getProductos (file:///C:/_discoNANCY/cursos/cursos_Coderhouse_backendNodejs/ProyectoFinal/backend/src/controller/webController.js:22:29)
    at Layer.handle [as handle_request] (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\layer.js:95:5)
    at C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\index.js:341:12)
    at next (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\index.js:174:3)
    at router (C:\_discoNANCY\cursos\cursos_Coderhouse_backendNodejs\ProyectoFinal\backend\node_modules\express\lib\router\index.js:47:12)
[2022-06-19T16:37:24.671] [ERROR] default - ReferenceError: title is not defined
    at getProductos (file:///C:/_discoNANCY/cursos/cursos_Coderhouse_backendNodejs/ProyectoFinal/backend/src/controller/webController.js:26:43)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[2022-06-19T17:24:58.423] [ERROR] default - CustomError {
  estado: 404,
  descripcion: 'Document not found in undefined with that {"id":"A003"}',
  detalles: undefined
}
[2022-06-19T20:22:00.808] [ERROR] default - Envio de mail fallo la verificacion del servidor Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials r9-20020a9d5cc9000000b0060b1f3924c3sm6691073oti.44 - gsmtp
[2022-06-19T20:32:30.325] [ERROR] default - Envio de mail fallo la verificacion del servidor Error: Invalid login: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials n9-20020a4ae1c9000000b0040edc685e11sm6770214oot.13 - gsmtp
[2022-06-22T17:00:59.302] [ERROR] default - MongoInvalidArgumentError: Query filter must be a plain object or ObjectId
    at new FindOperation (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\operations\find.js:19:19)
    at FindCursor._initialize (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\find_cursor.js:49:31)
    at next (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:454:16)
    at C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:177:13
    at maybePromise (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\utils.js:496:5)
    at FindCursor.next (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:173:41)
    at Collection.findOne (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\collection.js:233:76)
    at CarritosDao.getById (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/model/daos/ContainerDao.js:33:47)
    at CarritosApi.getProductosDelCarrito (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/api/CarritosApi.js:15:52)
    at obtenerProductosDelCarrito (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/controller/CarritosController.js:24:40)
[2022-06-22T17:00:59.305] [ERROR] default - CustomError {
  estado: 500,
  descripcion: 'Error when obtaining a Document by id in the collection carritos',
  detalles: MongoInvalidArgumentError: Query filter must be a plain object or ObjectId
      at new FindOperation (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\operations\find.js:19:19)
      at FindCursor._initialize (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\find_cursor.js:49:31)
      at next (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:454:16)
      at C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:177:13
      at maybePromise (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\utils.js:496:5)
      at FindCursor.next (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\cursor\abstract_cursor.js:173:41)
      at Collection.findOne (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\mongodb\lib\collection.js:233:76)
      at CarritosDao.getById (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/model/daos/ContainerDao.js:33:47)
      at CarritosApi.getProductosDelCarrito (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/api/CarritosApi.js:15:52)
      at obtenerProductosDelCarrito (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/controller/CarritosController.js:24:40)
}
[2022-06-22T17:01:43.108] [ERROR] default - CustomError {
  estado: 404,
  descripcion: 'Document not found in undefined with that {"id":"A003"}',
  detalles: undefined
}
[2022-06-22T17:29:19.545] [ERROR] default - ReferenceError: usuarios is not defined
    at Strategy._verify (file:///C:/_repositorios/nodeJS_TPFinal_backend/src/routes/usuarios.js:37:22)
    at Strategy.authenticate (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\passport-local\lib\strategy.js:90:12)
    at attempt (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\passport\lib\middleware\authenticate.js:369:16)
    at authenticate (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\passport\lib\middleware\authenticate.js:370:7)
    at Layer.handle [as handle_request] (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\layer.js:95:5)
    at C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\_repositorios\nodeJS_TPFinal_backend\node_modules\express\lib\router\index.js:341:12)
[2022-06-22T20:45:46.267] [ERROR] default - Falló el envio de mail del nuevo pedido - error:ReferenceError: err is not defined
[2022-06-22T20:51:57.784] [ERROR] default - Falló el envio de whatsapp del nuevo pedido - error:ReferenceError: err is not defined
[2022-06-22T20:54:29.560] [ERROR] default - Falló el envio de whatsapp del nuevo pedido - error:ReferenceError: err is not defined
[2022-06-22T21:01:39.255] [ERROR] default - Falló el envio de whatsapp del nuevo pedido - error:ReferenceError: err is not defined
[2022-06-22T23:27:09.183] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: enviarSMS is not defined
[2022-06-22T23:52:17.730] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
[2022-06-23T00:06:46.329] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
[2022-06-23T00:15:02.697] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
[2022-06-23T00:15:50.430] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
[2022-06-23T00:16:50.633] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
[2022-06-23T00:18:21.346] [ERROR] default - Falló el envio de SMS de confirmarción al usuario - error:ReferenceError: err is not defined
